guacamole-ext/src/main/java/org/apache/guacamole/net/auth/simple/SimpleConnection.java

```diff
@@ -53,6 +52,12 @@
      * Backing configuration, containing all sensitive information.
      */
     private GuacamoleConfiguration fullConfig;
+    
+    /**
+     * The proxy configuration used to tell the client how to connect to
```

Comment:
Same here - I'm iffy on the phrasing "to tell the client how to connect" given the context of this object.

Reply:
Fixed.